apps/ocp-plugin/src/components/AppContext/AppContext.tsx

```diff
@@ -42,15 +41,18 @@ const appRoutes = {
   [ROUTE.RESOURCE_SYNC_DETAILS]: '/edge/resourcesyncs',
   [ROUTE.ENROLLMENT_REQUESTS]: '/edge/enrollmentrequests',
   [ROUTE.ENROLLMENT_REQUEST_DETAILS]: '/edge/enrollmentrequests',
+  [ROUTE.COMMAND_LINE_TOOLS]: '/', // CLI downloads are shown embedded in OCP's CLI downloads page and not as an independent route
 };
 
 export const useValuesAppContext = (): AppContextProps => {
   const fetch = useFetch();
-  const metrics = useMetrics();
   const userInfo = useSelector(getUser);
 
   return {
     appType: FlightCtlApp.OCP,
+    settings: {
+      isRHEM: true,
```

Comment:
this should not be hardcoded, you can still run ocp-plugin as flightctl UI.

Do we even need this here ? OCP plugin never shows its branding anywhere

Reply:
We'd need to make the `isRHEM` prop optional if we want to prevent having to hardcode it.
Do you think it's better to do so ?

Reply:
yes, lets make it optional (and not set it for OCP)